src/components/AccessibilityPolicyForm/index.jsx

```diff
@@ -0,0 +1,278 @@
+                  }
+                  return errorMessage;
+                })}
+              </ul>
+            </div>
+          </div>
+        ),
+      };
+    } else if (accessibilityStatus.type ===
+        accessibilityActions.ACCESSIBILITY_FORM_SUBMIT_RATE_LIMIT_FAILURE) {
+      status = {
+        alertType: 'danger',
+        alertDialog: (
+          <div>
+            We are currently experiencing high volume. Try again later today or
+             send an email message to <a href={`mailto:${email}`}>{email}</a>
+          </div>
+        ),
+      };
+    } else if (accessibilityStatus.type ===
+        accessibilityActions.ACCESSIBILITY_FORM_SUBMIT_SUCCESS) {
+      status = {
+        alertType: 'success',
+        alertDialog: (
+          <div>
+            Thank you for contacting edX!
+            <br />
+            <br />
+            Thank you for your feedback regarding the accessibility of Studio. We typically respond
+             within one business day (Monday to Friday, 13:00 to 21:00 UTC).
+          </div>
+        ),
+      };
+    }
+
+    return status;
+  }
+
+  validateFormContent(inputType, inputValue) {
+    let feedback = { isValid: true };
+    if (inputType === 'email') {
+      /* eslint-disable max-len */
+      /* eslint-disable no-useless-escape */
+      const emailRegEx = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+      /* eslint-enable max-len */
+      if (!emailRegEx.test(inputValue)) {
+        feedback = {
+          isValid: false,
+          validationMessage: 'Valid email required.',
+        };
+      }
+    } else if (inputType === 'fullName') {
+      if (!inputValue) {
+        feedback = {
+          isValid: false,
+          validationMessage: 'Full name required.',
+        };
+      }
+    } else if (inputType === 'message') {
+      if (!inputValue) {
+        feedback = {
+          isValid: false,
+          validationMessage: 'Message required.',
+        };
+      }
+    }
+
+    return feedback;
+  }
+
+  closeStatusAlert() {
+    this.emailInputRef.focus();
+    this.setState({
+      statusAlertOpen: false,
+    });
+  }
+
+  handleChange(value, name) {
+    if (name === 'email') {
+      this.setState({
+        formEmail: value,
+      });
+    } else if (name === 'fullName') {
+      this.setState({
+        formFullName: value,
+      });
+    } else if (name === 'message') {
+      this.setState({
+        formMessage: value,
+      });
+    }
+  }
+
+  renderStatusAlert() {
+    const status = this.getStatusFields();
+
+    const statusAlert = (
+      <StatusAlert
+        alertType={status.alertType}
+        dialog={status.alertDialog}
+        open={this.state.statusAlertOpen}
+        onClose={this.closeStatusAlert}
+        ref={(input) => { this.statusAlertRef = input; }}
+      />
+    );
+
+    return (
+      <div>
+        {statusAlert}
```

Comment:
Why not skip the variable assignment and just return specify the `StatusAlert` component here? Do you need to wrap it in a `div` or can you just return it?

Reply:
The idea here is if there was going to be additional styling necessary around the status alert to be implemented. I'm going to leave this as is for now, but once we've confirmed the final styles if this is still as is, we can move it to avoid the extra declaration.